### PR TITLE
[FIX][PGIO-112] - TransactionModal component accessibility

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Description
* Implements a hidden `DialogTitle` and `DialogDescription` for the Transaction Modal a11y
* ~Temporary hides a11y warnings until we get a `DialogDescription` component exported from `@swapr/ui`, and we can create and hide an a11y compliant dialog description~

### CODE REVIEW

**Review by berteotti — APPROVED** [review]

File: app/components/TransactionModal.tsx [hunk 1]
```diff
@@ -34,7 +37,14 @@ export const TransactionModal = ({
   return (
     <Dialog open={isModalOpen(ModalId.WAITING_TRANSACTION)} onOpenChange={close}>
       <DialogContent>
-        <DialogHeader />
+        <DialogHeader>
+          <VisuallyHidden asChild>
+            <>
+              <DialogTitle>Transaction Modal</DialogTitle>
+              <DialogDescription />
```
Comment (Diogomartf) [inline app/components/TransactionModal.tsx:44]: do we need the `DialogDescription`?

**Review by Diogomartf — APPROVED** [review]

File: app/components/TransactionModal.tsx (hunk 1 above)
Comment (ElRodrigote) [reply app/components/TransactionModal.tsx:44]: Yes, both Title and Description are required by Content:
* The Title missing triggers an error severity console message
* The Description missing triggers a warning severity console message

Placing both components inside the Content DOM tree removes the error and the warning.
`VisuallyHidden` can only have 1 children, that's why I used a Fragment to wrap both the Title and the Description

File: app/components/TransactionModal.tsx [hunk 2]
```diff
@@ -34,7 +37,14 @@ export const TransactionModal = ({
   return (
     <Dialog open={isModalOpen(ModalId.WAITING_TRANSACTION)} onOpenChange={close}>
       <DialogContent>
-        <DialogHeader />
+        <DialogHeader>
+          <VisuallyHidden asChild>
+            <>
```
Comment (berteotti) [inline app/components/TransactionModal.tsx:42]: Do we need the Fragment?

Comment (ElRodrigote) [reply app/components/TransactionModal.tsx:42]: Yes :c

`VisuallyHidden` can only have 1 children, that's why I used a Fragment to wrap both the Title and the Description

**Pull request merged**